src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -257,6 +258,31 @@ public bool RunSettingsIsChecked
             }
         }
 
+        /// <summary>
+        /// Controls the Node autocomplete method Object type IsChecked property in the Features ObjectType radio button
+        /// </summary>
+        public bool NodeAutocompleteObjectTypeIsChecked
+        {
+            get
+            {
+                return preferenceSettings.DefaultNodeAutocompleteSuggestion == NodeAutocompleteSuggestion.ObjectType;                
+            }
+            set
+            {
+                if (value)
+                {
+                    preferenceSettings.DefaultNodeAutocompleteSuggestion = NodeAutocompleteSuggestion.ObjectType;
+                    nodeAutocompleteObjectTypeIsChecked = NodeAutocompleteSuggestion.ObjectType;
+                }
+                else
+                {
+                    preferenceSettings.DefaultNodeAutocompleteSuggestion = NodeAutocompleteSuggestion.MLRecommendation;
+                    nodeAutocompleteObjectTypeIsChecked = NodeAutocompleteSuggestion.ObjectType;
```

Comment:
This line is shared in `if` and `else`, maybe move it out to below?

Reply:
Fixed.